api/jobs/gears.py

```diff
@@ -0,0 +1,32 @@
+"""
+Gears
+"""
+
+from .. import config
+
+log = config.log
+
+
+def get_gears():
+    """
+    Fetch the install-global gears from the database
+    """
+
+    gear_doc  = config.db.static.find_one({'_id': 'gears'})
+    return gear_doc['gear_list']
```

Comment:
Could have error-checking around a missing/malformed gears doc but that would be overkill and the 500 that will get logged for the KeyError will make the actual error clear enough.